hangouts_chat.py

```diff
@@ -221,7 +221,13 @@ def _subscribe_to_pubsub_topic(self, project, topic_name, subscription_name, cal
         return subscriber.subscribe(subscription_name, callback=callback)
 
     def _handle_message(self, message):
-        data = json.loads(message.data.decode('utf-8'))
+        try:
```

Comment:
Can we move this below the `if not data.get('message'):` so we don't try and parse the JSON if it's `None`?

Reply:
You want to parse the JSON *after* we use it? Either that doesn't compute or I am not getting it :) The 'message' in `data.get('message')` is from the actual JSON payload while the `message` in `message.data` is from PUB/SUB and is (supposedly) guaranteed to be there.

Reply:
Gah. Don't mind me. I'm being silly.